Mathlib/Tactic/RelCongr/Core.lean

```diff
@@ -265,61 +265,59 @@ partial def _root_.Lean.MVarId.relCongr
   -- to record which arguments to the head function differ between the two sides of the goal.
   let varyingArgs := tplArgs.map (·.2)
   if varyingArgs.all not then
-    throwError "try refl"
+    throwError "try rfl"
```

Comment:
even better: you could generate a clickable suggestion with `Std.Tactic.TryThis.addSuggestion`

Reply:
I would prefer not to since there is no guarantee that `rfl` will actually work here -- the suggestion is made whether or not the relation is actually reflexive.  (Actually, I preferred not to have a special case here at all, but Mario thinks it might be helpful.)